addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -23,12 +24,18 @@ export class DiscussCoreCommon {
         this.threadService = services["mail.thread"];
     }
 
+    /** @returns {import("models").Thread} */
+    insertInitChannel(data) {
+        return this.createChannelThread(data);
+    }
+
     setup() {
         this.messagingService.isReady.then((data) => {
-            for (const channelData of data.channels) {
-                this.createChannelThread(channelData);
-            }
-            this.threadService.sortChannels();
+            Record.MAKE_UPDATE(() => {
```

Comment:
I understand the case that `MAKE_UPDATE` is static so it is all capital. But it feels weird to call a function like that outside the `Record` file. If this is the case, then it might be a good idea to provide a public interface without the all-capital function name.

Reply:
I just put capital letters to put the code a bit more in change, as there's already a lot of static methods.
This function allow to wrap an update transaction, so it's fine to have it readily available in business code sometimes.